docs/GET_STARTED.md

```diff
@@ -50,20 +50,22 @@ client.on("ready", () => {
     console.log("Hello world!");
     const dashboard = new Dashboard(client, {
         port: 8080, // Dashboard port? Number, default: 8080
-        clientSecret: "Your discord bot client secret", 
+        clientSecret: "Your discord bot client secret",
         redirectURI: "http://localhost:8080/auth/discord/callback",
         maintenanceNotification: false, // Maintenance notification? Boolean, default: false
-        baseGame: '', // Base game? String, default: !help | v0.0.6.3,
-        baseBotStatus: '', // Bot status? String, default: online
-        maintenanceGame: '', // Maintenance Game? String, default: Bot is in maintenance
-        maintenanceStatus: '', // Maintenance Status? String, default: dnd
-        sessionSecret: 'a random text (I recommend punching your keyboard))', // Session Secret? String, default: abcdefghijklmnopqrstuvwxyz1234567890!@#$%¨&*()_ABCDEFGHIJKLMNOPQRSTUVWXYZ
+        baseGame: "", // Base game? String, default: !help | v0.0.6.3,
+        baseBotStatus: "", // Bot status? String, default: online
+        maintenanceGame: "", // Maintenance Game? String, default: Bot is in maintenance
+        maintenanceStatus: "", // Maintenance Status? String, default: dnd
+        sessionSecret: "a random text (I recommend punching your keyboard))", // Session Secret? String, default: abcdefghijklmnopqrstuvwxyz1234567890!@#$%¨&*()_ABCDEFGHIJKLMNOPQRSTUVWXYZ
+        websiteDomain: "http://localhost:8080",
```

Comment:
Just gonna state it here because the code pertaining to this is all over, the bot shouldn't deal with any environment variables itself. You have no guarantee the user isn't already using them for something else and didn't mean to pass them. Also, it abstracts the configuration, and requires us to maintain more than needed. (In terms of docs and making sure that the variables never change across the entire code base.)